core/src/main/java/com/kin/ecosystem/core/util/DateUtil.java

```diff
@@ -13,8 +13,10 @@
     private static DateFormat utcDateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm", Locale.US);
     private static TimeZone utcTimeZone = TimeZone.getTimeZone("UTC");
 
+    private static DateFormat utcDateFormatExtended = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'", Locale.US);
 	static {
 		utcDateFormat.setTimeZone(utcTimeZone);
+        utcDateFormatExtended.setTimeZone(utcTimeZone);
```

Comment:
Is this intentionally indented this way?

Reply:
github issue somehow